src/Nethermind/Nethermind.Consensus/MiningConfig.cs

```diff
@@ -42,8 +45,16 @@ public string ExtraData
             }
             set
             {
+                byte[] bytes = Encoding.UTF8.GetBytes(value);
+                if (bytes != null && bytes.Length > 32)
+                {
+                    throw new InvalidConfigurationException($"Extra Data length was more than 32 bytes. You provided: {_extraDataString}",
```

Comment:
Did you check if this exception will stop the node?

Reply:
Yes:
in Debug
![image](https://user-images.githubusercontent.com/2915361/197521635-cbc50699-1875-4150-8c21-b1670a77d786.png)
in Release
![image](https://user-images.githubusercontent.com/2915361/197521845-a9cd6bfe-ffc8-4833-8b68-1b69742d89f7.png)

Reply:
Would prefer a nicer message, not an exception trace.
It is now thrown very early in the pipeline and not handled correctly or it is because it is being packed by other exceptions. I also think exit code might not be used. Can we make this simple error log in next release?